2996_reflection/reflection.md

```diff
@@ -1877,10 +1947,8 @@ This paper is proposing that:
 
 ### Freestanding implementations
 
-
-Several important metafunctions, such as `std::meta::_nonstatic_data_members_of`, return a `std::vector` value.
-Unfortunately, that means that they are currently not usable in a freestanding environment.
-That is an highly undesirable limitation that we believe should be addressed by imbuing freestanding implementations with a more restricted `std::vector` (e.g., one that can only allocate at compile time).
+Several important metafunctions, such as `std::meta::nonstatic_data_members_of`, return a `std::vector` value.
+Unfortunately, that means that they are currently not usable in a freestanding environment, but a recent paper ([@P3295R0]) currently being seen by SG7 proposes freestanding `std::vector`, `std::string`, and `std::allocator` in constant evaluated contexts, explicitly to make the facilities proposed by this paper work in freestanding.
```

Comment:
I think it's enough to just point out the paper - the phrase "currently being seen by SG7" isn't true yet, and will only have been true for a very short period of time (if at all... probably it should go straight to LEWG?) 

```suggestion
Unfortunately, that means that they are currently not usable in a freestanding environment, but [@P3295R0] currently proposes freestanding `std::vector`, `std::string`, and `std::allocator` in constant evaluated contexts, explicitly to make the facilities proposed by this paper work in freestanding.
```